types/robust-segment-intersect/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        
+    }
+}
```

Comment:
The `package.json` file shouldn't need to exist unless there are dependencies declared

Reply:
thank you

Reply:
@MysteryBlokHed is right -- this package.json needs to be removed since these types don't have any dependencies.

Reply:
The file should be deleted since the Definitely Typed publisher will be able to generate a correct one itself.